core/src/main/scala/uk/gov/nationalarchives/tdr/api/core/graphql/service/FileService.scala

```diff
@@ -0,0 +1,44 @@
+package uk.gov.nationalarchives.tdr.api.core.graphql.service
+
+import uk.gov.nationalarchives.tdr.api.core
+import uk.gov.nationalarchives.tdr.api.core.File
+import uk.gov.nationalarchives.tdr.api.core.db.dao.{ConsignmentDao, FileDao}
+import uk.gov.nationalarchives.tdr.api.core.db.model.{ConsignmentRow, FileRow}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class FileService(fileDao: FileDao, consignmentService: ConsignmentService)(implicit val executionContext: ExecutionContext) {
+
+  def all: Future[Seq[File]] = {
+    fileDao.all.flatMap(fileRows => {
+      val files = fileRows.map(fileRow =>
+        consignmentService.get(fileRow.consignmentId).map(consignment =>
```

Comment:
Just FYI, I know this is an inefficient way of fetching related entities - we have to make a separate DB query for each consignment, and yet another for the series. There's a comment in the SeriesService about fixing it - Sangria has something called a deferred resolver, which might help.

I wonder if we should look into it soon, since it's going to be a general issue with GraphQL requests since they have the option of returning very deeply nested objects?